Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```
@@ -642,3 +642,60 @@
   - type: Tag
     tags:
     - Meat
+
+- type: entity
+  parent: FoodMealBase
+  id: FoodMeatSausage
+  name: cooked sausage
+  description: A simple sausage. Who knows what's in this thing?
+  components:
+  - type: Tag
+    tags:
+    - Cooked
+    - NFHotdog
+  - type: FlavorProfile
+    flavors:
+      - meat
```

```suggestion
      - meaty
```
Minor spelling mistake (doesn't give the flavor when eating it)